pkg/templates/cluster_run_template.go

```diff
@@ -31,18 +31,19 @@ type Outputs map[string]apiextensionsv1.JSON
 type ClusterRunTemplate interface {
 	GetName() string
 	GetResourceTemplate() v1alpha1.TemplateSpec
-	GetOutput(stampedObjects []*unstructured.Unstructured) (Outputs, error)
+	GetOutput(stampedObjects []*unstructured.Unstructured) (Outputs, *unstructured.Unstructured, error)
```

Comment:
nit: not everyone agrees with this, _but_, for interfaces, I find useful to name the returns so that it's less ambiguous at a first look (in a codebase with little comments) to understand what's returned